### ISSUE TITLE
Add only specific files

### ISSUE THREAD

**Issue by Jiayou-Chao** [mufeedvh/code2prompt#21]
Can I just add specific files like `code2prompt add file1,file2`? This feature would be great.

**Comment by ODAncona** [comment]
@Jiayou-Chao I implemented this feature in #20 using glob pattern, thus providing a similar api as `tar` or `rsync`.

You can use it like this :

```sh
code2prompt path/to/codebase --include="*.rs,*.toml, **/rel/path/to/file1.png"
```

**Comment by Jiayou-Chao** [comment]
Thanks. This is even better.

**Comment by mufeedvh** [comment]
Fixed in #20 by @ODAncona! 🙌 🎉
